.travis.yml

```diff
@@ -9,7 +9,6 @@ env:
 language: python
 python:
   - 2.7
```

Comment:
If you remove this, does travis get rid of the "Python 2.7" entries? If so I think that would be better because as-is it is a tad misleading.